Example/Auth/ApiTests/AnonymousAuthTests.m

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2019 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import <XCTest/XCTest.h>
+
+#import "FIRAuthApiTestsBase.h"
+
+@interface AnonymousAuthTests : FIRAuthApiTestsBase
+
```

Comment:
remove the new line here.

Reply:
You mean line 22 right? I prefer keep it since it's makes the class structure clearer

Reply:
Removing the line for a simple class with 0 method is more common among Google3 code. see http://google.github.io/styleguide/objcguide.html#prefixes

It's ok if you dont remove, because I don't care much about it.